code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -341,8 +341,8 @@
 	item_state = "lasercarbine[magazine ? "-[CEILING(get_ammo(0)/5, 1)*5]" : ""]"
 
 /obj/item/gun/projectile/automatic/laserrifle
-	name = "security laser rifle"
-	desc = "A bulky, single barreled rifle that uses disposable laser cartridges rather than an internal power cell. Utilized by Nanotrasen's private security force."
+	name = "IK-M1 laser carbine"
+	desc = "A bulky, Warp-Tec Industries single barreled rifle that uses disposable laser cartridges rather than an internal power cell. Sold to Nanotrasen's private security forces."
```

Comment:
```suggestion
	desc = "A sleek, Warp-Tac Industries laser rifle that uses disposable laser cartridges rather than an internal power cell. Sold to Nanotrasen's private security forces."
```